components/camera.js

```diff
@@ -1,212 +1,23 @@
-import * as React from 'react';
-import { Button, Image, View } from 'react-native';
-import * as ImagePicker from 'expo-image-picker';
-import Constants from 'expo-constants';
-import * as Permissions from 'expo-permissions';
-import  { useState, useEffect } from 'react';
-import { Text, TouchableOpacity } from 'react-native';
-import { Camera } from 'expo-camera';
-import { Ionicons } from '@expo/vector-icons';
-import { useIsFocused} from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack';
+import * as React from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { CameraScreen } from "./cameraScreen";
+import { PicturePage } from "./picturePage";
```

Comment:
A best practice is to implement consistent naming. If you name one component `CameraScreen`, the other should be `PictureScreen`

Reply:
ok boomer